lib/pharos/host/ubuntu/ubuntu_bionic.rb

```diff
@@ -56,6 +56,22 @@ def configure_container_runtime
           raise Pharos::Error, "Unknown container runtime: #{host.container_runtime}"
         end
       end
+
```

Comment:
Both Xenial and Bionic have the exact same logic, could this method come from their common super class?

Reply:
Fixed.